R/combine_cleaning_logs.R

```diff
@@ -44,7 +44,8 @@
 
    print(names(list_of_log) |> glue::glue_collapse(" ,")  %>% glue::glue("List of element to combine- ", .))
 
```

Comment:
glue::glue_collapse(", ") instead of (" ,") for correct punctuation
![image](https://github.com/impact-initiatives/cleaningtools/assets/79831444/794ba587-3cd6-4558-9593-5dfbb9c4240f)